js/src/lib/interfaces/Libraries.ts

```diff
@@ -239,12 +240,14 @@ model = joblib.load(
 	}
 };
 
-
 const fastai = (model: ModelData) =>
 	`from huggingface_hub import from_pretrained_fastai
 
 learn = from_pretrained_fastai("${model.id}")`;
 
+const sampleFactory = (model: ModelData) =>
+	`python -m sample_factory.huggingface.load_from_hub -r ${model.id} -d <train_dir_path>`;
```

Comment:
Instead of `-d <train_dir_path>` I think we should have `-d ./train_dir` which is the default, just so users can copy paste this directly

Reply:
✔️